.yarnrc.yml

```diff
@@ -1,6 +1,7 @@
 logFilters:
-  # react-native-web overrides react-native, so we don't need to have react-native installed
   - level: discard
     text: "em@workspace:. doesn't provide react-native (p52384), requested by react-native-gesture-handler."
 
 nodeLinker: node-modules
+
+yarnPath: .yarn/releases/yarn-4.5.1.cjs
```

Comment:
Can you explain this and the addition of the release file?

Reply:
Ah, this thing is still showing up even though I switched to yarn 4! Annoying

Reply:
This issue seems to happen when an older version of yarn is still installed despite having the newest version running.